### DISCUSSION TITLE
What is the status of the `Team review assignments` preview is the GraphQL API?

### DISCUSSION THREAD

**Discussion by felixlut** [community/community#72860] [Enterprise]
### `Team review assignments` preview

What is the status of the ``Team review assignments`` preview is the GraphQL API? 

### Body

I recently started playing around with using the [Terraform GitHub provider](https://github.com/integrations/terraform-provider-github) to manage GitHub teams in my Organization, including the [Code Review settings](https://docs.github.com/en/enterprise-cloud@latest/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team). The provider does support controlling these settings via the [`github_team_settings`](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/team_settings) resource, which relies on the GraphQL API to function. The specific endpoints it's using are the ones listed [here](https://docs.github.com/en/enterprise-cloud@latest/graphql/overview/schema-previews#team-review-assignments-preview), where it's specified that they are in preview. 

My question now is what the status of the preview is? It seems like it's been in preview since at least `2020-01-14`, which is when the `Team review assignments preview` was added to the docs of [Enterprise Server 2.20](https://docs.github.com/en/enterprise-server@2.20/graphql/overview/schema-previews#team-review-assignments-preview) (it's nowhere to be found on [the same page for 2.19](https://docs.github.com/en/enterprise-server@2.19/graphql/overview/schema-previews#team-review-assignments-preview)). 

The reason I'm asking is because I realized that there are some features missing from the [`github_team_settings`](https://registry.terraform.io/providers/integrations/github/latest/docs/resources/team_settings) resource, which cannot be added since they are not available in the GraphQL API (see https://github.com/integrations/terraform-provider-github/issues/1972 for reference).

**Comment by ghostinhershell** [comment]
Hey @felixlut, thanks so much for reaching out in our community 👋🏾 

So I have some good news here! I had an engineer take a look at this and it appears that the feature you are looking for here is already implemented and just seems to be blocked by the preview status. They went ahead and opened a PR to remove the preview status and hopefully that should unblock you here. 

Now I do want to note that [GitHub Universe](https://githubuniverse.com/) is happening this week so you won't hear any updates during this time but hopefully this will gain traction again next week. Let me know if you have any additional questions in the meantime and, if you did plan on attending Universe whether virtually or in person, check out the conversations that will be happening under [the Universe 2023 heading in the Community](https://github.com/orgs/community/discussions/categories/ai) 🪐 

Cheers!

**Reply by felixlut** [reply]
Hey @ghostinhershell! 

I can see that ``Team review assignments`` is no longer in preview, but the feature I want is still nowhere to be found. To be clear, what I want is the possibility to read the exclusion list for the team review assignments, as described in https://github.com/integrations/terraform-provider-github/issues/1972. `excludedTeamMemberIds` is available in the [`updateTeamReviewAssignment`](https://docs.github.com/en/graphql/reference/mutations#updateteamreviewassignment) mutation endpoint, so it's possible to update the field. But it's not possible to read it, or at least I haven't been able to find it when reading the docs for querying [GitHub teams](https://docs.github.com/en/graphql/reference/objects#team) via the GraphQL API, even though the other review settings are there (eg, `reviewRequestDelegationAlgorithm`)

**Comment by nunocenteio** [comment]
Any update on this?
